src/lib/Injector.java

```diff
@@ -0,0 +1,38 @@
+package lib;
+
+import controller.ConsoleHandler;
+import dao.BetDaoImpl;
+import dao.HumanDaoImpl;
+import factory.BetDaoFactory;
+import factory.HumanDaoFactory;
+
+import java.lang.reflect.Field;
+
+public class Injector {
+
+    public static void injectDependency() throws IllegalAccessException {
+        Class<ConsoleHandler> consoleHandlerClass = ConsoleHandler.class;
+        Class<BetDaoImpl> betDaoImplClass = BetDaoImpl.class;
+        Class<HumanDaoImpl> humanDaoClass = HumanDaoImpl.class;
+
+        Field[] consoleHandlerFields = consoleHandlerClass.getDeclaredFields();
+
+        for (Field field : consoleHandlerFields) {
+            String typeField = field.getType().toString();
+
+            if (typeField.equalsIgnoreCase("interface dao.BetDao")
```

Comment:
better:
field.getType().equals(BetDao.class)

Reply:
ok, I agree with you. I forgot how I get a class.